docs/index.rst

```diff
@@ -0,0 +1,71 @@
+.. Images
+.. |slack_1| image:: images/slack_01.jpg
```

Comment:
Are you happy to show publicly your web hook url? You can blur out the service id part

Reply:
It's a fake one - I changed a few characters before taking the screen dump - but thanks for thinking about it.